src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
+
+  stylesArray.forEach((style) => {
+    const [key, value] = style.split(':').map((item) => item.trim());
 
+    if (key && value) {
+      stylesObject[key] = value;
+    }
+  });
```

Comment:
The chaining of 'split' and 'map' functions is well done. However, ensure that the code is consistently indented to improve readability, especially when chaining methods. In this case, the indentation is consistent, but keep this in mind for future reference.